examples/tcp_listenfd_server.rs

```diff
@@ -19,10 +19,8 @@ const DATA: &[u8] = b"Hello world!\n";
 
 #[cfg(not(windows))]
 fn get_first_listen_fd_listener() -> Option<std::net::TcpListener> {
-    #[cfg(unix)]
-    use std::os::unix::io::FromRawFd;
-    #[cfg(target_os = "wasi")]
-    use std::os::wasi::io::FromRawFd;
+    #[cfg(any(unix, target_os = "wasi"))]
+    use std::os::fd::FromRawFd;
```

Comment:
This module is rather new, so this change may conflict with decreasing the MSRV as discussed in #1732. Maybe just introduce your own module for this functionality?

Reply:
`std::os::fd` is from Rust 1.66, in #1749 we'll need a MSRV of 1.71 (?), so this won't make the difference.

Reply:
Now we also have Hermit, this makes the implementations a lot simpler.